packages/typegen/src/util/derived.ts

```diff
@@ -58,7 +58,7 @@ export function getSimilarTypes (registry: Registry, definitions: Record<string,
         );
 
         possibleTypes.push(`([${subs.join(', ')}])[]`);
-      } else {
+      } else if (subDef.info !== TypeDefInfo.VecFixed) {
```

Comment:
```suggestion
      } else if (subDef.info === TypeDefInfo.VecFixed) {
        // TODO: Add possibleTypes so imports work
      } else {
```

Reply:
This will need to be looked at specifically - the `possibleTypes` contains the imports as well. So if already imported, it will be all ok. Additionally it adjust the signature. So for now, just marked as a TODO, since it probably will need some love.